harness/determined/deploy/aws/templates/lore.yaml

```diff
@@ -0,0 +1,1084 @@
+                startup_script: |
+                  mkdir -p /mnt/efs
+                  mount -t efs ${EFSVal}:/ /mnt/efs
+                  if [ ! -z "${DockerUser}" ] && [ ! -z ${DockerPass} ]; then
+                    echo "${DockerPass}" | docker login --username "${DockerUser}" --password-stdin
+                  fi
+
+            scheme: $scheme
+            cpu_env_image: ${CpuEnvImage}
+            gpu_env_image: ${GpuEnvImage}
+            notebook_timeout: ${NotebookTimeout}
+            lore_version: ${LoreVersion}
+
+            bind_mounts:
+              - host_path: /mnt/efs
+                container_path: /run/determined/workdir/shared_fs
+            EOF
+
+            apt-get remove docker docker-engine docker.io containerd runc
+            apt-get update
+            apt-get install -y \
+                apt-transport-https \
+                ca-certificates \
+                curl \
+                gnupg-agent \
+                software-properties-common
+            curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+            add-apt-repository \
+               "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+               $(lsb_release -cs) \
+               stable"
+            apt-get update
+            apt-get install -y docker-ce docker-ce-cli containerd.io
+
+            curl -fsSL https://www.amazontrust.com/repository/AmazonRootCA1.pem > \
+              /usr/local/determined/etc/db_ssl_root_cert.pem
+
+            if [ ! -z "${DockerUser}" ] && [ ! -z ${DockerPass} ]; then
+              echo "${DockerPass}" | docker login --username "${DockerUser}" --password-stdin
+            fi
+
+            docker network create determined
+
+            docker stop $(docker ps -a -q)
+            docker rm $(docker ps -a -q)
+
+            mkdir -p /mnt/efs
+
+            date >> /tmp/cloud-init-log
+
+            apt-get install -y binutils git
+            if [ ! -d "efs-utils" ]; then
+              git clone https://github.com/aws/efs-utils
+              cd efs-utils
+              ./build-deb.sh
+              chown _apt:root ./build/amazon-efs-utils*deb
+              apt-get install -y ./build/amazon-efs-utils*deb
+              cd ..
+              rm -r efs-utils
+            fi
+
+            mount -t efs ${EFSVal}:/ /mnt/efs
+
+            if [ ! -z "${LoreVersion}" ]; then
+              docker run -d \
+                --name lore-backend \
+                --env DET_MASTER=http://determined-master:8080 \
+                --env DB_NAME=lore \
+                --env DB_USER=postgres \
+                --env DB_PASSWORD=${DBPassword} \
+                --env DB_HOST=${Database.Endpoint.Address} \
+                --env DB_PORT=5432 \
+                -p 9012:9012 \
+                -v /mnt/efs:/run/determined/workdir/shared_fs \
+                --network determined \
+                "determinedai/environments-dev:lore-backend-image-${LoreVersion}"
```

Comment:
should this be `${ImageRepoPrefix}/${MasterImageName}...`?

Reply:
if we want to add another flag to the cli we can. we shouldn't need to change this but we can add it for more fleixiblity